server.R

```diff
@@ -112,8 +112,9 @@ server <- function(input, output) {
   )
   
   
+  # no longer need for this
   # mapa RO
-  rondonia <- jsonlite::read_json("https://raw.githubusercontent.com/luizpedone/municipal-brazilian-geodata/master/data/RO.json")
+  #rondonia <- jsonlite::read_json("https://raw.githubusercontent.com/luizpedone/municipal-brazilian-geodata/master/data/RO.json")
   
   rondonia$features <- rondonia$features %>%
     purrr::map(function(x){

```

Comment:
The e_map_register_ui(...) has generated the map but without tooltip like before.
How can I adjust the tooltip without object 'rondonia'?